web/package.json

```diff
@@ -8,7 +8,8 @@
     "build": "npm run scaffold && NODE_ENV=production webpack -p",
     "lint": "eslint 'src/**/*.js'",
     "start": "webpack-dev-server --host 0.0.0.0 --port 8001 --history-api-fallback --content-base src/",
-    "test": "jest",
+    "test": "jest --collectCoverage",
+    "test-watch": "jest --watch",
```

Comment:
I could add another docker container that runs jest and lint as it detects changes.  🤔  (It'd probably be an awful experience.)